### PR TITLE
feat: created login and signup page

### PR DESCRIPTION
**Pull request by nakulsharma15**
Feat: Created login and signup pages

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -28,7 +28,7 @@ body {
 }
 
 .main-body {
-    width: "80%";
+    width: 80%;
     display: "flex";
```
Comment (vaibhav-mahalle) [inline src/App.css:32]: you could use utility class here,

**Review by vaibhav-mahalle — APPROVED** [review]
just a small PR not much to review you can proceed further.

File: src/Pages/Login.jsx [hunk 2]
```diff
@@ -1,8 +1,51 @@
+import { Header, Footer, Aside } from "../Components/index";
```
Comment (coldpigli) [inline src/Pages/Login.jsx:1]: can use default imports

**Review by coldpigli — APPROVED** [review]
One minor suggestion, rest everything looks good.

File: src/Pages/Login.jsx (hunk 2 above)
Comment (nakulsharma15) [reply src/Pages/Login.jsx:1]: Thank you for the suggestion. I'll look into it.

File: src/App.css (hunk 1 above)
Comment (nakulsharma15) [reply src/App.css:32]: I'll look into this. Thanks for pointing out.

**Pull request merged**